gmso/utils/files/gmso_xmls/test_ffstyles/oplsaa-ethane_foyer.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I think we can rename this file to be just `oplsaa_ethane.xml`

Reply:
I agree the name isn't great here, it was copied from the gmso testing suite. I changed the name to oplsaa_from_foyer.xml to better reflect it's origin and what we're testing for with it. There's already an alkanes.xml that should be the same as the oplsaa_ethane.xml example we would want to cover.